packages/communication-react/CHANGELOG.stable.md

```diff
@@ -10,25 +10,22 @@ Mon, 10 Feb 2025 18:17:45 GMT
 [Compare changes](https://github.com/azure/communication-ui-library/compare/1.24.0-beta.1...1.24.0)
```

Comment:
Change link to: https://github.com/azure/communication-ui-library/compare/1.23.0...1.24.0